airflow/models/baseoperator.py

```diff
@@ -933,7 +938,7 @@ def __init__(
         if dag:
             self.dag = dag
 
-        self._log = logging.getLogger("airflow.task.operators")
+        self._log = logging.getLogger(logger_name) if logger_name else None
```

Comment:
Empty string and None mean different things for `getLogger`, this should allow both.

Reply:
Indeed, will fix and add a test for that. Thanks !